doc/policy_tutorial/POLICY_RUN_LOCALLY.md

```diff
@@ -0,0 +1,149 @@
+# Run the policy locally
+
+Now that the development and unit testing of the policy is completed it is time to actually start APIcast with the newly created policy. This way we can verify the policy actually works inside the Nginx process. We are again going to use the APIcast development Docker image to run APIcast with our newly created policy.
+
+## Create APIcast configuration
+
+APIcast can be provisioned in two ways, either automatically from the 3scale API Manager or via a json configuration file. Since the custom policy is not available in the 3scale API Manager we need to configure APIcast with the json configuration file. In this configuration file we are also going to leverage the build-in echo service of APIcast so we can test the policy and receive an upstream response without running a full 3scale API Manager configuration.
+
+The part of the json configuration file detailing the configuration of our policy need to adhere to the json schema we defined earlier.
+
+The location of the config file is somewhat arbitrary as long as the APIcast process can access the file. In this example we are going to create the following file **‘hello_world_config.json’** in the **apicast/examples/configuration** directory.
+
+The contents of the configuration file is:
+
+```json
+{
+  "services": [
+    {
+      "proxy": {
+        "policy_chain": [
+          {
+            "name": "hello_world",
+            "version": "builtin",
+            "configuration": {
+              "overwrite": true,
+              "secret": "mysecret"
+            }
+          },
+          {
+            "name": "apicast.policy.upstream",
+            "configuration": {
+              "rules": [
+                {
+                  "regex": "/",
+                  "url": "http://echo:8081"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+Inside the policy chain we first configure our hello_world policy with the overwrite and secret properties. Second in the policy chain is the upstream policy which acts as an echo mock service so we can actually receive a response.
+
+### Starting the APIcast server
+Now in order to start the APIcast server with the hello_world_configuration.json file inside the development container issue the following command:
+
+```shell
+bash-4.2$ bin/apicast --log-level=debug --dev -c examples/configuration/hello_world_config.json
```

Comment:
I will suggest to use the following: 
```
THREESCALE_CONFIG_FILE=hello_world_config.json APICAST_LOG_LEVEL=debug ./bin/apicast
```

The main reason is how the people will use env variables, so things are easy to test in that way.